src/app/home-page/widget/organization-box/organization-box.component.html

```diff
@@ -0,0 +1,137 @@
+<mat-card class="h-100 org-box">
+  <div id="header-bar" fxLayout="row" fxLayoutGap="1rem" class="mat-card-header-separate">
+    <div id="organization-number" fxLayout="row">
+      <mat-card-header fxLayout="row">
+        <mat-card-title>Organizations</mat-card-title>
+        <span class="bubble">{{ listOfOrganizations.length }}</span>
+      </mat-card-header>
+    </div>
+    <div fxLayout="row" fxLayoutGap="0.5rem">
+      <div id="pending-requests" fxLayout="row" fxLayoutAlign="center center" class="mb-3">
+        <mat-icon class="pending-requests mr-1">pending_icon</mat-icon>
+        <a routerLink="/accounts" class="size-small pending-links">
+          <div [ngPlural]="pendingRequests.length">
+            <ng-template ngPluralCase="=0">0 Pending Requests</ng-template>
+            <ng-template ngPluralCase="=1">1 Pending Request</ng-template>
+            <ng-template ngPluralCase="few">{{ pendingRequests.length }} Pending Requests</ng-template>
+          </div>
+        </a>
+      </div>
+      <div id="pending-invites" fxLayout="row" fxLayoutAlign="center center" class="mb-3">
+        <mat-icon class="pending-invites mr-1">error</mat-icon>
+        <a routerLink="/accounts" class="size-small pending-links">
+          <div [ngPlural]="pendingInvites.length">
+            <ng-template ngPluralCase="=0">0 Pending Invitations</ng-template>
+            <ng-template ngPluralCase="=1">1 Pending Invitation</ng-template>
+            <ng-template ngPluralCase="few">{{ pendingInvites.length }} Pending Invitations</ng-template>
+          </div>
+        </a>
+      </div>
+      <button
+        mat-button
+        class="private-btn small-btn-structure mb-3"
+        id="requestOrganizationButton"
+        matTooltip="Request Organization"
+        matTooltipPosition="after"
+        (click)="requireAccounts()"
+      >
+        <img src="../../../assets/svg/icons-plus.svg" alt="Register an Organization" class="pr-2 pb-1" />
+        Request an Organization
+      </button>
+    </div>
+  </div>
+  <mat-divider></mat-divider>
+  <mat-card-content class="p-3 h-100">
+    <div *ngIf="events.length !== 0 || isLoading; else noEvents">
+      <h5 class="weight-bold">Recent Activity</h5>
+      <div class="size-small list-events" *ngFor="let event of events" fxLayout="row" fxLayoutAlign="space-between center">
+        <div fxLayout="row" fxLayoutAlign="start center">
+          <img src="{{ event?.initiatorUser?.avatarUrl }}" alt="User avatar" />
+          <div [ngSwitch]="event.type">
+            <div *ngSwitchCase="EventType.CREATECOLLECTION" class="my-3">
+              <strong>{{ event.initiatorUser?.username }}</strong> created the collection
+              <a [routerLink]="'/organizations/' + event.organization?.name + '/collections/' + event.collection?.name"
+                >{{ event.collection?.name }}
```

Comment:
The whitespace at the end of this line renders as a trailing space in the collection link, you can move the `</a>` so it's directly after the text (no whitespace between) to eliminate it.